### PR TITLE
Basic Containers and Pill

### PR DESCRIPTION
**Pull request by kennethlien**
Row, background container, and `Pill` components for usage throughout app.

### CODE REVIEW

**Comment by kennethlien** [comment]
image

File: src/components/Components.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    borderRadius: 100,
+    borderWidth: 1,
+    borderColor: Colors.Grey100,
+    padding: 10,
+    alignItems: 'center',
+  },
+});
+
+export function ScreenContainer(props: React.ComponentProps<typeof View>) {
+  return <View {...props} style={[styles.screen, props.style]} />;
```
Comment (mylesdomingo) [inline src/components/Components.tsx:23]: what is this used for?

File: src/components/Components.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+export function RowContainer(props: React.ComponentProps<typeof View>) {
```
Comment (mylesdomingo) [inline src/components/Components.tsx:26]: im fine with creating layout components for rows and columns,  but put it in the shared components folder as `Row.tsx`

File: src/components/Components.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+const defaultPillProps = {
+  value: '',
+  icon: '',
+};
+export function Pill(props: PillProps & React.ComponentProps<typeof View>) {
+  return (
+    <RowContainer style={[styles.pill, props.style]}>
+      {props.icon !== '' && (
+        <FontAwesome
```
Comment (mylesdomingo) [inline src/components/Components.tsx:43]: Use he default text component

**Review by mylesdomingo — CHANGES_REQUESTED** [review]

**Review by nikkisuzani — COMMENTED** [review]
lgtm

**Pull request closed without merging**